<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LessonDialogClass</class>
 <widget class="QDialog" name="LessonDialogClass">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LessonDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item row="4" column="3">
    <widget class="QPushButton" name="decline_button">
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTimeEdit" name="time_edit">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QComboBox" name="weekday_combo">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Monday</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tuesday</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Wednesday</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Thursday</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Friday</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Saturday</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sunday</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QPushButton" name="accept_button">
     <property name="text">
      <string>OK</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLineEdit" name="address_edit">
     <property name="placeholderText">
      <string>Address...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="4" colspan="2">
    <widget class="QFrame" name="children_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item row="2" column="1">
       <widget class="QLineEdit" name="children_surname">
        <property name="placeholderText">
         <string>Child surname...</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="children_name">
        <property name="placeholderText">
         <string>Child name...</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QScrollArea" name="children_scroll">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="children_scroll_contents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>485</width>
           <height>304</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout"/>
        </widget>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="children_label">
        <property name="text">
         <string>Children:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QFrame" name="employee_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item row="1" column="0" colspan="3">
       <widget class="QScrollArea" name="employee_scroll">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="employee_scroll_contents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>235</width>
           <height>181</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2"/>
        </widget>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="employee_label">
        <property name="text">
         <string>Employees:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="employee_surname">
        <property name="placeholderText">
         <string>Employee surname...</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="employee_name">
        <property name="placeholderText">
         <string>Employee name...</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>accept_button</sender>
   <signal>clicked()</signal>
   <receiver>LessonDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>decline_button</sender>
   <signal>clicked()</signal>
   <receiver>LessonDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
